version.go

```diff
@@ -23,4 +23,4 @@ package tchannel
 // VersionInfo identifies the version of the TChannel library.
 // Due to lack of proper package management, this version string will
 // be maintained manually.
-const VersionInfo = "0.0.1-dev"
+const VersionInfo = "1.0.0"
```

Comment:
Typically, yes. Some of the build tools, e.g. Glide, also allow pinning to a tag.